### PR TITLE
bring toolbar dropdown styles inline

### PR DESCRIPTION
**Pull request by rgbkrk**
💥 

Another component finished off with `styled-jsx` adaptation.

### CODE REVIEW

File: packages/core/src/components/toolbar.js [hunk 1]
```diff
@@ -188,7 +188,7 @@ export default class Toolbar extends PureComponent<ToolbarProps> {
           }
 
           .cell-toolbar {
-            background-color: var(--toolbar-bg);
+            background-color: var(--toolbar-bg, rgba(255, 255, 255, 0.5));
```
Comment (rgbkrk) [inline packages/core/src/components/toolbar.js:191]: I also included some missing defaults from #2069.

**Comment by captainsafia** [comment]
Nice! I gave AppVeyor a wangle-jangle to get the funky-gunky out of the way. Once that is green, this is good to go!

**Comment by rgbkrk** [comment]
> I gave AppVeyor a wangle-jangle to get the funky-gunky out of the way.

wangle-jangle is my guiding phrase for today

I'm happy to rebase this directly since it's likely conflicts in the `main.css`.

**Comment by captainsafia** [comment]
> I'm happy to rebase this directly since it's likely conflicts in the main.css.

I fixed the conflicts unless you're referencing something else.

**Comment by rgbkrk** [comment]
On github, for me, it's still showing `This branch cannot be rebased due to conflicts`. 😦

**Comment by rgbkrk** [comment]
Guess squashing is probably fine though, my default UI just shows that mergability is `nil`.

**Pull request merged**